src/System.Private.Xml/src/System/Xml/XPath/Internal/XPathArrayIterator.cs

```diff
@@ -84,7 +84,5 @@ public override IEnumerator GetEnumerator()
         {
             return list.GetEnumerator();
         }
-
-        private object debuggerDisplayProxy { get { return index < 1 ? null : (object)new XPathNavigator.DebuggerDisplayProxy(Current); } }
```

Comment:
Sorry I meant to add to my main message that the merge should be done by someone who knows what they are doing, hence why I left in such things to ensure that it would flag as a bad merge